app/models/service_template_ansible_playbook.rb

```diff
@@ -151,4 +151,17 @@ def update_catalog_item(options, auth_user = nil)
     end
     super
   end
+
+  def destroy(auth_user = 'system')
+    config_info = options[:config_info]
+    [:provision, :retirement, :reconfigure].each do |action|
```

Comment:
@bzwei - Currently all CURD instance methods are following this pattern.  I would rather adjust the instance CRUD methods in one single way via `resource_action.configuration_template` via a followup PR.